src/styles/AdminDashboardHome/index.less

```diff
@@ -36,6 +38,20 @@ h2 {
   margin-left: 10px;
 }
 
+.see-all-link {
+  color: white;
+  background-color: #21C5B5;
+  padding: 0px .5rem 0px .5rem;
+  border-radius: .15rem;
+  flex-shrink: 0;
+  width: 4.8rem;
+  font-size: 1rem;
+}
+
+.see-all-link:hover {
+  color: black;
```

Comment:
lets find out if we'll need these styles for any other links within the app? If we do then we can consider styling out our links globally within the `themeOverrides.js` file.